docs/faq.rst

```diff
@@ -115,6 +122,34 @@ See above.
 
 ------
 
+**Q. Why do I see error code 503 with the message “Error: configuration operation in progress on device, Please try again in 2 min” on the 'Pending tasks' table of the app?**
+
+If a BIG-IP is already processing an AS3 asynchronous task, it displays this message for any successive async operations. Once the asynchronous operation is completed, it is possible to perform the next operation like create/update/delete partition or application. Please wait for a few minutes and try the configuration again. 
+
+------
+
+**Q. For all L4-L7 App Services operations, why do I see the message “BIG-IP is processing the request. Please click the 'Pending Tasks' icon to check the status of the pending request.”?**
+
+All the L4-L7 App Services are processed in an asynchronous manner in the background. Hence this message is displayed for every operation done on this tab. The “Pending Tasks” icon shows the status of last few such pending requests and their status. Also, once any such pending task is completed on the BIG-IP the UI reloads the data to display the latest AS3 information about partitions and applications. 
```

Comment:
minus "Hence" and start with "This..."
Pending Tasks should probably be bold and not quoted, so The **Pending Tasks** icon shows...
"Also, once any such pending task is completed on the BIG-IP, the UI reloads..."  note the comma.